common/models/product.json

```diff
@@ -33,10 +36,16 @@
       "description": "the listing wineries"
     },
     "grapes": {
-      "type": "hasAndBelongsToMany",
+      "type": "referencesMany",
       "model": "Grape",
       "foreignKey": "grapeIds",
-      "description": "the listing types of grapes"
+      "description": "the types of grapes contained in this wine"
+    },
+    "region": {
```

Comment:
i haven't create relation between Winery and Region yet. so right now, they are just separate relations of Product.